_posts/2024-02-12-introducing-kimai-mobile-time-tracking-features.md

```diff
@@ -0,0 +1,59 @@
+
+![Kimai Mobile Multilingual-Support]({% link /images/news/Kimai-Mobile-Multilingual-Support.png %}){:class="image"}
+
+### Dark mode: Elevate style and Comfort
+
+For those with a keen eye for design, Kimai Mobile's Dark Mode offers a stylish solution worth exploring. Easily switch between the classic light interface and the sleek dark theme. This feature not only enhances your visual comfort but also provides a personalized touch to your time tracking journey.
+
+![Kimai Mobile DarkMode]({% link /images/news/Kimai-Mobile-DarkMode.png %}){:class="image"}
+
+### Track anywhere with offline mode 
+
+Say bye to connectivity concerns, as Offline Mode allows you to log your work hours without the need for a constant internet connection. Whether you're on a flight, in remote locations, or facing internet issues, experience uninterrupted time tracking with the Kimai Mobile App. This feature provides you with the flexibility of tracking your time anytime, anywhere! 
+
+![Kimai Mobile Offline]({% link /images/news/Kimai-Mobile-Offline.png %}){:class="image"}
+
+### Take a break: Mindfulness for enhanced productivity 
+
+In case you forget, Kimai Mobile reminds you to prioritize your mental health and productivity. Through friendly notifications, the app encourages taking timely breaks, allowing you to stay revitalized and maintain peak efficiency throughout your day 
+
+![Kimai Mobile Break]({% link /images/news/Kimai-Mobile-Break.png %}){:class="image"}
+
+### Data-driven insights with comprehensive reports 
+
+The statistical analysis offers a comprehensive overview of your time usage across different projects. Tailor your experience further by customizing reports to specific time frames, empowering you to make informed decisions based on detailed information. 
+
+![Kimai Mobile Reports]({% link /images/news/Kimai-Mobile-Reports.png %}){:class="image"}
+
+### In conclusion
+
+In summary, Kimai Mobile goes beyond a simple time tracking app; it's designed to help you manage your time effectively. It offers features like multilingual support, Dark Mode, Offline Mode, Break reminders, and detailed reports, providing tools for efficient time tracking and increased productivity. 
+
+Take the chance to improve how you manage time and boost productivity on the go. Try out Kimai Mobile today to experience a positive change in your work life. 
+
+![Kimai Mobile Download Now]({% link /images/news/Kimai-Mobile-DownloadNow.png %}){:class="image"}
+
+Ready to take control of your time? Start using Kimai Mobile and discover efficient time tracking for yourself. 
+
+### Download Kimai Mobile now:
+
+[Google Play]({% link /_store/cloudrizon-kimai-mobile-app-android.md %})
+[App Store]({% link /_store/cloudrizon-kimai-mobile-app-ios.md %})
+
+For in-depth information about Kimai Mobile and its features, visit: [Kimai Mobile Features](https://www.kimaimobile.com/features/)
```

Comment:
in-depth information belong into the store page, where you can add a link to your own domain, but not here.
remove this line please